### DISCUSSION TITLE
How to get story format highlighting?

### DISCUSSION THREAD

**Discussion by loopernow** [cyrusfirheir/twee3-language-tools#47] [Q&A] [answered]
Hello! I am working with Chapbook 1.2.1 and I'm wondering how to get the Chapbook highlighting as shown in the documentation. For me, the documentation is a little unclear.

I think it's partially working in my file. My StoryData passage looks like this:

`:: StoryData
{
        "ifid": "49845A7E-9CCE-472B-BE13-113245797E01",
        "format": "Chapbook",
	"format-version": "1"
}`

I don't know if that's right!

The variables section of a passage seems to be highlighted mostly correctly (even if the colors are not quite the same colors as the screenshot in the documentation). Variable names, string values, and numeric values are all different colors. But Boolean values are not highlighted. They are the exact same color as the variable names (dark blue). This is what leads me to believe I may be doing something wrong.

**Comment by cyrusfirheir** [comment, marked as answer]
A `StoryData` like this should work fine:
```json
:: StoryData
{
  "ifid": "49845A7E-9CCE-472B-BE13-113245797E01",
  "format": "Chapbook",
  "format-version": "1.2.1"
}
```
Note the *full* semantic version `1.2.1`, instead of just the major version `1`. (Though it shouldn't break anything, your version. It'd still detect properly.)

As for the colors, the screenshots are taken with [another theme](https://marketplace.visualstudio.com/items?itemName=tobiasalthoff.atom-material-theme) and a bit customized, at that. The default VSCode Dark+ theme will look different.

**Reply by loopernow** [reply]
Excellent. Thanks so much. Switched themes and, yep, the color differentiation is better. :)
